### PR TITLE
Improve C++ header documentation

### PR DESCRIPTION
**Pull request by sipa**

### CODE REVIEW

File: include/minisketch.h [hunk 1]
```diff
@@ -214,55 +221,70 @@ class Minisketch
...
+    /** Check whether this Minisketch object is valid. */
     explicit operator bool() const noexcept { return bool{m_minisketch}; }
 
+    /** Construct an (invalid) Minisketch object. */
     Minisketch() noexcept = default;
+
+    /** Move constructor. */
     Minisketch(Minisketch&&) noexcept = default;
+
+    /** Move assigningment. */
```
Comment (gmaxwell) [inline include/minisketch.h:233]: assignment

File: include/minisketch.h [hunk 2]
```diff
@@ -186,19 +186,26 @@ class Minisketch
...
+     *  being decoded incorrectly (and will instead fail when decoding for up to n elements).
+     *
+     *  See minisketch_compute_capacity for more details. */
     static size_t ComputeCapacity(uint32_t bits, size_t max_elements, uint32_t fpbits) noexcept { return minisketch_compute_capacity(bits, max_elements, fpbits); }
 
-    /** See minisketch_compute_max_elements(). */
+    /** Reverse operation of ComputeCapacity. See minisketch_compute_max_elements. */
     static size_t ComputeMaxElements(uint32_t bits, size_t capacity, uint32_t fpbits) noexcept { return minisketch_compute_max_elements(bits, capacity, fpbits); }
 
     /** Construct a clone of the specified sketch. */
```
Comment (jnewbery) [inline include/minisketch.h:211]: Maybe document that this function will abort if the provided minisketch is invalid. Same for the copy assignment below.

**Review by jnewbery — COMMENTED** [review]
Looks good. One suggestion inline.

File: include/minisketch.h (hunk 1 above)
Comment (sipa) [reply include/minisketch.h]: Fixed.

File: include/minisketch.h (hunk 2 above)
Comment (sipa) [reply include/minisketch.h:211]: Nice catch. I've instead added a commit that makes it work as expected when passing an invalid object.

**Comment by gmaxwell** [comment]
utack db0cade615f492ad43f2caba3f6a9ddcbb00e8bd

**Pull request merged**